File_handling/fileOperations.test.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs')
+
+const {persons, saveDataToFile} = require('./fileOperations.js')
+
+test('Reading data from file', () => {
+    const readMe = 'persons.json'
+    const testData = fs.readFileSync(readMe, 'utf-8')
+    const expectedData = JSON.parse(testData)
+    expect(expectedData).toStrictEqual(persons)
+})
+
+test('Saving data to file', () => {
+    const readMe2 = 'persons2.json'
+    let persons2 = []
+    fs.writeFileSync(readMe2, JSON.stringify(persons2, null, 2), 'utf-8');
+    const fileData = fs.readFileSync(readMe2, 'utf-8')
```

Comment:
You dont need this test because you are testing only the native fa framework itself, not something that you created yourself, get it?

Reply:
I understand... but in the file created for utility functions (fileOperations.js), there are only two functions using the fs framework... in this case, would it be unnecessary to create these tests?